uni/android/app/proguard-rules.pro

```diff
@@ -30,3 +30,4 @@
 
 -dontwarn com.google.errorprone.annotations.*
 -dontwarn javax.annotation.*
+-dontwarn javax.annotation.concurrent.*
```

Comment:
Isn't this included in the `javax.annotation.*`? Or does that not apply recursively?

Reply:
it does not iterate recursively